src/autoDeleteMessages.ts

```diff
@@ -1,23 +1,53 @@
 import { BotContext } from './context';
 
-export interface AutoDeleteMessage {
-  [messageId: number]: number;
-}
+const expired = (createdAt: number, timeout: number) => {
+  const minute = 60000;
+  return Math.floor((Date.now() - createdAt) / minute) >= timeout;
+};
 
 export const runMessageRecycling = async (
   ctx: BotContext
 ): Promise<void> => {
-  const database = ctx.database;
+  const database = await ctx.loadDatabase();
   const messages = database.getAutoDeleteMessages();
-  const minute = 60000;
-  const recycleTimeout = 1;
-
-  Object.entries(messages).map(async ([id, createdAt]) => {
-    if (
-      Math.floor((Date.now() - createdAt) / minute) >= recycleTimeout
-    ) {
-      await ctx.deleteMessage(+id);
-      await database.removeAutoDeleteMessage(+id);
+  const messageTimeout = 1;
+
+  Object.entries(messages).map(async ([unsafeId, createdAt]) => {
+    if (!expired(createdAt, messageTimeout)) {
+      return;
+    }
+
+    const id = Number(unsafeId);
+
+    try {
+      ctx.logger.info(`Trying to delete message`, {
+        id,
+        chatId: ctx.chat?.id,
+      });
```

Comment:
I need to fix this, this was a test on giving args to the logger message but didn't work out

Reply:
fixed